Documentation/CRDs/Shared-Filesystem/ceph-filesystem-crd.md

```diff
@@ -133,15 +133,18 @@ spec:
     enabled: true
     # list of Kubernetes Secrets containing the peer token
     # for more details see: https://docs.ceph.com/en/latest/dev/cephfs-mirroring/#bootstrap-peers
+    # Add the secret name if it already exists else specify the empty list here.
     peers:
       secretNames:
-        - secondary-cluster-peer
```

Comment:
If the list is empty IIRC you need `{}` else the empty list may give an invalid syntax.

Reply:
@travisn when i mentioned it just like that without `{}` I didn't observe any error in my testing.

Reply:
Ok, what if we leave the example secret name commented out? It can be helpful to see an example
```
#- secondary-cluster-peer
```

Reply:
sure sounds good.